compiler/test/input/memoryBase/asserts.gr

```diff
@@ -3,9 +3,12 @@ module Asserts
 
 include "runtime/unsafe/wasmi32"
 
-primitive heapBase: WasmI32 = "@heap.base"
-primitive heapStart: WasmI32 = "@heap.start"
-primitive typeMetadata: WasmI32 = "@heap.type_metadata"
+@unsafe
+primitive heapBase = "@heap.base"
+@unsafe
+primitive heapStart = "@heap.start"
+@unsafe
+primitive typeMetadata = "@heap.type_metadata"
```

Comment:
I thought the changes in the compiler was adding the `disable_gc` attribute to these, so why do we still need `@unsafe`? I'm probably not following it correctly.

Reply:
I actually removed disable_gc from them because it made it pretty complicated. I liked this a lot better because they're unsafe values, and like anything else, they shouldn't be allowed outside of unsafe contexts.